suite/spdz/src/test/java/dk/alexandra/fresco/suite/spdz/TestSpdzMisc.java

```diff
@@ -0,0 +1,49 @@
+package dk.alexandra.fresco.suite.spdz;
+
+import dk.alexandra.fresco.framework.MPCException;
+import dk.alexandra.fresco.framework.sce.resources.storage.FilebasedStreamedStorageImpl;
+import dk.alexandra.fresco.framework.sce.resources.storage.InMemoryStorage;
+import dk.alexandra.fresco.suite.spdz.configuration.PreprocessingStrategy;
+import dk.alexandra.fresco.suite.spdz.storage.DataSupplierImpl;
+import dk.alexandra.fresco.suite.spdz.storage.DummyDataSupplierImpl;
+import dk.alexandra.fresco.suite.spdz.storage.SpdzStorage;
+import dk.alexandra.fresco.suite.spdz.storage.SpdzStorageImpl;
+import dk.alexandra.fresco.suite.spdz.utils.Util;
+import java.security.NoSuchAlgorithmException;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Tests miscellaneous classes such as util and resource pool
+ *
+ */
+public class TestSpdzMisc {
+
+  @Test
+  public void testUtil() {
+    Util util = new Util();
+    Assert.assertNotNull(util);
```

Comment:
Odd test. Maybe just delete the Util class. It does not seem to do anything.

Reply:
It holds the exp pipe number. Magic number which I would love to get rid off, but seems not that easy to do properly.

Reply:
Ok, but couldn't that number be moved somewhere more appropriate? If we keep the class anyway, then at least it should not be possible to instantiate, as in the this test.